bower.json

```diff
@@ -1,39 +1,39 @@
 {
-    "name": "uber",
-    "version": "14.0.0",
-    "homepage": "https://github.com/magfest/ubersystem",
-    "authors": [
-        "Eli Courtwright <eli@courtwright.org>"
-    ],
-    "description": "MAGFest Ubersystem",
-    "main": "uber/templates/base.html",
-    "license": "BSD",
-    "ignore": [
-        "**/.*",
-        "node_modules",
-        "bower_components",
-        "test",
-        "tests"
-    ],
-    "dependencies": {
-        "lodash": "3.8.0",
-        "jquery": "2.1.4",
-        "jquery-ui": "1.11.4",
-        "bootstrap": "3.2.0",
-        "moment": "2.10.2",
-        "jquery-hoverIntent": "1.8.1",
-        "select2": "4.0.0",
-        "jQuery-contextMenu": "1.6.6",
-        "jquery-confirm": "2.3.1",
-        "toastr": "2.1.1",
-        "footable": "2.0.3",
-        "geocomplete": "1.6.5",
-        "angular": "1.3.15",
-        "angular-route": "1.3.15",
-        "angular-bootstrap": "0.13.0"
-    },
-    "resolutions": {
-        "jquery": ">= 1.9.0",
-        "bootstrap": "3.2.0"
-    }
```

Comment:
Dunno why the indentation changed all of a sudden, but 2 spaces is probably more standard for json files anyway, I suppose.